src/send_kcidb.py

```diff
@@ -0,0 +1,109 @@
+#!/usr/bin/env python3
+#
+# SPDX-License-Identifier: LGPL-2.1-or-later
+#
+# Copyright (C) 2022 Maryam Yusuf
+# Author: Maryam Yusuf <maryam.m.yusuf1802@gmail.com>
+import datetime
+import os
+import sys
+
+import kernelci
+import kernelci.data
+from kernelci.config import load
+from kernelci.cli import Args, Command, parse_opts
+from kcidb import Client
+import kcidb
+
+
+class cmd_run(Command):
+    help = "Listen for events and send them to KCDIB"
+    args = [Args.db_config]
+
+    def __call__(self, configs, args):
+        db_config = configs['db_configs'][args.db_config]
+        api_token = os.getenv('API_TOKEN')
+        db = kernelci.data.get_db(db_config, api_token)
+
+        project_id = os.getenv('KCIDB_PROJECT_ID')
+        topic_name = os.getenv('KCIDB_TOPIC_NAME')
+        google_credential = os.getenv('GOOGLE_APPLICATION_CREDENTIALS')
+        client = None
+
+        if topic_name and project_id:
+            client = Client(project_id=project_id,
+                            topic_name=topic_name)
+
+        sub_id = db.subscribe('node')
+        print("Listening for events... ")
+        print("Press Ctrl-C to stop.")
+        sys.stdout.flush()
+        if not client or not google_credential:
+            print("Aborting due to missing configuration")
+            sys.stdout.flush()
+            return
```

Comment:
Please move this before calling `db.subscribe()` or it'll leak a pub/sub subscription.

Reply:
I have moved it